psychopy/parallel/__init__.py

```diff
@@ -39,21 +39,22 @@
     from ._linux import PParallelLinux
     ParallelPort = PParallelLinux
 elif sys.platform == 'win32':
-    drivers = dict(inpout32=['_inpout', 'PParallelInpOut'],
-                   inpoutx64=['_inpout', 'PParallelInpOut'],
-                   dlportio=['_dlportio', 'PParallelDLPortIO'])
+    drivers = dict(inpout32=('_inpout', 'PParallelInpOut'),
+                   inpoutx64=('_inpout', 'PParallelInpOut'),
+                   dlportio=('_dlportio', 'PParallelDLPortIO'))
     from ctypes import windll
     from importlib import import_module
     for key, val in drivers.items():
+        driver_name, class_name = val
         try:
             hasattr(windll, key)
-            import_module('.'+val[0], __name__)
-            ParallelPort = getattr(locals()[val[0]], val[1])
+            import_module('.'+driver_name, __name__)
+            ParallelPort = getattr(eval(driver_name), class_name)
```

Comment:
Just checked the `importlib` docs; `import_module()` returns the loaded module, so we should be able to do:
```python
ParallelPort = getattr(import_module('.' + driver_name, __name__),
                       class_name)
```
if I'm not mistaken? WDYT?